src/js/labels/labels.js

```diff
@@ -0,0 +1,20 @@
+export const verbs = {
+    PLAY: () => 'play',
+    PAUSE: () => 'pause',
+    MUTE: () => 'mute',
+    UNMUTE: () => 'unmute',
+    ENTER_FULLSCREEN: () => 'enter fullscreen mode',
+    EXIT_FULLSCREEN: () => 'exit fullscreen mode',
+    ENTER_PIP: () => 'enter picture in picture mode',
+    EXIT_PIP: () => 'exit picture in picture mode',
+    SEEK_FORWARD_N_SECS: ({ seekOffset }) => `seek forward ${seekOffset} seconds`,
+    SEEK_BACK_N_SECS: ({ seekOffset }) => `seek back ${seekOffset} seconds`,
+};
+
+
+export const nouns = {
```

Comment:
Just interested, what's the benefit of differentiating nouns from verbs here?

Reply:
was aiming for a bigger picture semantics (since nouns make sense for certain UI/a11y metaphors and verbs make sense for others), but happy to remove since we can easily re-add it or implement it in whatever way makes sense longer term (e.g. if/when we support a11y+i18n and if we end up going with well-defined format string/labeling standards like the ICU message syntax, used by CLDR JSON data https://formatjs.io/docs/core-concepts/icu-syntax/ or similar)